server/src/notification.js

```diff
@@ -0,0 +1,24 @@
+import fetch from 'node-fetch';
+
+const FIREBASE_SERVER_KEY = "AIzaSyBNjq8QTFCVLRYs9EyXa5UZjmJdSEPFxjA";
```

Comment:
You weren't supposed to commit this one :P
Can you rotate the key and set it up to use dotenv or something else where I can inject it into the environment in heroku?

Reply:
yeah I can, but as I said these keys are fairly ephemeral as that project can be murdered